lt/cmd/speech/main.go

```diff
@@ -83,3 +112,84 @@ func main() {
 
 	wg.Wait()
 }
+
+func performScript(filename string) error {
+	awsSess := session.Must(session.NewSessionWithOptions(session.Options{
+		SharedConfigState: session.SharedConfigEnable,
+	}))
+	svc := polly.New(awsSess)
+
+	f, err := os.Open(filename)
+	if err != nil {
+		log.Fatalf("open script %s failed: %v", filename, err)
+	}
+
+	script, err := importScript(f)
+	if err != nil {
+		log.Fatalf("parsing script %s failed: %v", filename, err)
+	}
+
+	stopCh := make(chan struct{})
+	var userWg sync.WaitGroup
+
+	var userClients []*client.User
+	for i, name := range script.users {
+		user := client.NewUser(client.Config{
+			Username:     name,
+			Password:     userPassword,
+			ChannelID:    channelID,
+			SiteURL:      siteURL,
+			WsURL:        wsURL,
+			Duration:     duration,
+			Speak:        true,
+			Setup:        setup,
+			TeamID:       teamID,
+			PollySession: svc,
+			PollyVoiceID: aws.String(script.voiceIds[i]),
+		})
+		userClients = append(userClients, user)
+
+		userWg.Add(1)
+		go func() {
+			defer userWg.Done()
+			if err := user.Connect(stopCh); err != nil {
+				log.Fatalf("connectUser failed: %s", err.Error())
+			}
+		}()
+	}
+
+	// "Conversation" logic
+	go func() {
+		time.Sleep(2 * time.Second) // time to take a sip of coffee before we talk talk talk
```

Comment:
Can we also send unmute events? Otherwise it feels a bit weird to hear sound when the UI is not matching.